docs/.vuepress/config.js

```diff
@@ -43,6 +43,25 @@ module.exports = {
         'getting-started',
         'core-functionality',
         'api',
+        {
+          title: 'Modules',
```

Comment:
@clevinson My idea here was to have a `Modules` section on the left, and in each module subsection, documentation about that module.

Let me know if you had something else in mind.